src/mumble/MainWindow.cpp

```diff
@@ -840,7 +840,8 @@ bool MainWindow::handleSpecialContextMenu(const QUrl &url, const QPoint &pos_, b
 			}
 		} else {
 			QByteArray qbaServerDigest = QByteArray::fromBase64(url.path().remove(0, 1).toLatin1());
-			cuContextUser              = ClientUser::get(url.host().toInt(&ok, 10));
+			QString id                 = url.host().split(".").value(1, "-1");
```

Comment:
Maybe we should add a small comment here, explaining what format we expect the url's host to be in.